app/src/test/java/org/simple/clinic/summary/PatientSummaryScreenControllerTest.kt

```diff
@@ -715,10 +766,14 @@ class PatientSummaryScreenControllerTest {
   }
 
   @Test
-  fun `when screen is opened and a canceled appointment with the patient does not exist then update phone dialog should not be shown`() {
```

Comment:
It took me a LOT of time to realize that you changed the ordering of these tests. The diff shows that many random changes were made to many functions and I was trying very hard to wrap my head around this.

Reply:
I actually just moved the `appointment cancelation reasons()` params provider method right below the test method where it was being used.